api/index.js

```diff
@@ -124,10 +134,16 @@ const createMeasurement = async (req, res, client) => {
         indexer_result,
         miner_id,
         provider_id,
+        alternative_provider_check_status_code,
+        alternative_provider_check_timeout,
+        alternative_provider_check_car_too_large,
+        alternative_provider_check_end_at,
+        alternative_provider_check_protocol,
+        alternative_provider_check_provider_id,
```

Comment:
Throwing this out there: We could alternatively implement this in such a way that after the alternative provider check has been completed, two measurements will have been created. One would link to the other. This would save us from having to duplicate the measurement schema inside itself. I don't think this is worth it yet though

Reply:
No that suggestion is really not important since spark-api's business is just buffering measurements until it flushes them again. If anything, we should discuss this in a repo that's further down the data processing pipeline

Reply:
I was thinking about this a bit and you are right. What started of as adding one field (status code for alternative retrieval) turned up into duplicating code a lot. We might be better off with adding a relationship between columns in the measurements table between regular and alternative provider check. That way we could avoid duplicating code down the processing and evaluation pipeline.